starter-code/app.js

```diff
@@ -1,5 +1,36 @@
+'use strict';
+
 const express = require('express');
 const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
 
+//Configure app
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+
+
+
+//Routes
+// retrieve a random chuck joke
+app.get('/', (req, res, next) => {
+    console.log('Get Working');
+    res.render('index');
+});
+
+app.get('/random', (req, res, next) => {
+    client.getRandomJoke()
+    .then((chuckRes) => {
+        // use the response here
+        res.render('index', chuckRes);
```

Comment:
chuckRes is an object and it has more things that the one you want to use. You can do something like:
```
const data = {
  joke: chuckRes.value
}
res.render('index', data);
```
Then you will acces from the view with:
```
<%= joke %>
```
Of course, your solution also works, but be careful. If the response is huge you are wasting resources sending everything when you only need a part.